docs/asciidoc/http-client.adoc

```diff
@@ -367,3 +368,69 @@ include::{examplesdir}/uds/Application.java[lines=18..33]
 ----
 <1> Specifies `DomainSocketAddress` that will be used
 ====
+== Host Name Resolution
+The `TCP` client uses by default `Netty's` domain name lookup mechanism that resolves a domain name asynchronously,
+this is as an alternative of the JVM's built-in blocking resolver.
+
+Below is the listing of the available configurations.
+
+[width="100%",options="header"]
+|=======
+| configuration name | description
+| cacheMaxTimeToLive | The max time to live of the cached DNS resource records (resolution: seconds).
+If the time to live of the DNS resource record returned by the DNS server is greater
+than this max time to live, this resolver will ignore the time to live from
+the DNS server and will use this max time to live.
+Default to Integer.MAX_VALUE.
+| cacheMinTimeToLive | The min time to live of the cached DNS resource records (resolution: seconds).
+If the time to live of the DNS resource record returned by the DNS server is less
+than this min time to live, this resolver will ignore the time to live from
+the DNS server and will use this min time to live.
+Default to 0.
+| cacheNegativeTimeToLive | the time to live of the cache for the failed DNS queries (resolution: seconds). Default to 0.
```

Comment:
Add backticks around `cacheNegativeTimeToLive`

Capitalize "The" at the start of the text in the next cell.

Change "Default to 0." to "Default: 0."